### PR TITLE
Datasource docs for Loki

### LINKED ISSUE #15436: Document using Loki in Grafana
Similar to all other core datasources in Grafana we need to write a "Using Loki in Grafana" documentation, similar to for example Prometheus (http://docs.grafana.org/features/datasources/prometheus/).

### PR DESCRIPTION
**Pull request by davkal**
- adds Loki data source docs to Grafana docs
- moved query-related docs from Explore to Loki

Fixes #15436

### CODE REVIEW

File: docs/sources/features/datasources/loki.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+## Templating
+
+Template variables are not yet supported by Loki.
+
+## Annotations
+
+Annotations are not yet supported by Loki.
+
+## Configure the Datasource with Provisioning
```
Comment (marefr) [inline docs/sources/features/datasources/loki.md]: Probably a good idea to use the basic auth example from here https://github.com/grafana/loki/blob/master/docs/grafana-provisioning.md#configuring-grafana-via-provisioning

And adding `maxLines` to that.

Comment (marefr) [reply docs/sources/features/datasources/loki.md]: Maybe you can change the loki getting started to point to grafana docs then instead.

**Comment by marefr** [comment]
Besides my minor comment think it looks great :+1:

File: docs/sources/features/datasources/loki.md (hunk 1 above)
Comment (davkal) [reply docs/sources/features/datasources/loki.md]: I would leave the example as is, since k8s seem more likely, and they dont rely on basic auth, AFAIK.

**Pull request merged**